start_sprint.sh

```diff
@@ -31,18 +31,19 @@ else
     exit 1
 fi
 
-cd "${SPRINTNAME}"
+cd "${SPRINTNAME}/drupal8"
```

Comment:
Should we go back to `${SPRINTNAME}` after it's finished so that a user could run start_sprint.sh again via up arrow (say if anything else fails)?

Reply:
IMO they want to be in the drupal8 directory. I don't actually understand why the sprint directory isn't a d8 checkout, not sure why there's a drupal8 underneath it, but I didn't want to make a lot of changes.  I always want to be in the project root when working on anything.

Reply:
If I'm in drupal8, and something went wrong and I wanted to start_sprint.sh, then I'd need to change back a directory. I think it might be confusing.

Reply:
start_sprint.sh would be run from ~/sprint, not from the sprint directory. You're remembering how start_clean.sh worked before.

Reply:
It's also worth just teaching people how to use ddev, and I expect that's what will happen. If "something goes wrong" they ought to be able to recover, between git and ddev. Anyway... still listening!

Reply:
I think we should be careful about that -- we don't want to cross the line between quickly enabling contributions and getting in the way to the point of needing to learn ddev (_especially_ if there's not a ddev person on-site).

Reply:
Yeah, I wasn't saying that we should require ddev training for everybody, just that starting over from scratch is not what I would be hoping to do as a sprint participant in the land of so-much opportunity to fix things.